app/components/deprecation-item-source.js

```diff
@@ -1,26 +1,38 @@
-import { tagName } from '@ember-decorators/component';
-import { computed, get } from '@ember/object';
+import { get, action } from '@ember/object';
 import { inject as service } from '@ember/service';
-import { and, readOnly, bool } from '@ember/object/computed';
-import Component from '@ember/component';
 
-@tagName('')
+import Component from '@glimmer/component';
 export default class DeprecationItemSource extends Component {
   @service port;
 
-  @bool('model.map.source') known;
-
-  @computed('model.map.{line,source}', 'known')
   get url() {
-    let source = get(this, 'model.map.source');
+    let source = this.itemModel.map.source;
     if (this.known) {
-      return `${source}:${get(this, 'model.map.line')}`;
+      return `${source}:${this.args.itemModel.map.line}`;
     } else {
       return 'Unkown source';
     }
   }
 
-  @readOnly('port.adapter') adapter;
+  get adapter() {
+    return this.port.adapter;
+  }
 
-  @and('known', 'adapter.canOpenResource') isClickable;
+  get isClickable() {
+    return this.known && this.adapter.canOpenResource;
```

Comment:
Since we are removing computeds, do we need to make `known` etc `@tracked`?

Reply:
Hm, I don't think so because `known` and `adapter` are getters.

Reply:
We don't need to make it tracked. The tracked property should be defined in this component instead of derived from args. And it should be mutable by this component. For example, `@tracked known = true` then we have an action which can mutate this property `this.known = false`. 
However in our use case, known is computed by getter and it is automatically tracked.
```  
  get known() {
    return this.args.itemModel.map.source;
  }
```